src/formatDate.js

```diff
@@ -50,7 +50,59 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const data = date.split(fromFormat[3]);
+  let fourNumbers = true;
+
+  let year;
+  let month;
+  let day;
+  let separator;
+
+  const answer = [];
+  let finalFormat = 0;
+
+  for (const time of fromFormat) {
+    if (time === 'YY') {
+      year = data[fromFormat.indexOf(time)];
```

Comment:
Could you move `data[fromFormat.indexOf(time)]` to a variable? You use it no once